cmd/Dockerfile

```diff
@@ -5,18 +5,24 @@ FROM golang:1.15-buster AS build-setup
 
 RUN apt-get update
 RUN apt-get -y install cmake zip sudo
+# add the pubkey of github.com to knownhosts, so ssh-agent doesn't bark
+RUN mkdir -p /root/.ssh && ssh-keyscan -t rsa github.com >> /root/.ssh/known_hosts
+RUN git config --global 'url.ssh://git@github.com/.insteadof' https://github.com/
```

Comment:
I assume these changes are safe to be backported @vishalchangrani

Reply:
this is needed only if we are using the private cadence repo and on master we are not (as per the go.mod below). So we don't need these two lines of changes and can be reverted to the version on master

Reply:
ok going to remove these lines